indico/modules/attachments/models/folders.py

```diff
@@ -91,6 +91,14 @@ def __table_args__(cls):
         default=True
     )
 
+    #: If the folder is never shown in the frontend (even if you can access it)
+    is_always_hidden_in_fe = db.Column(
```

Comment:
I kind of dislike this name. It's not very clear without reading the comment, and also too verbose. IMO `is_hidden` is a much better name for it. (also, we don't put an empty line between column definitions, and only one empty line between columns and relationships)

Reply:
Changed to is_hidden_in_fe .

Reply:
I think the `_in_fe` is not very useful either; that's something worth mentioning in the docstring on top of it (like it already does)

Reply:
Ok, using is_hidden now